### ISSUE TITLE
Signal_connect doesn't seem to work with Julia-1.0.0 on Linux

### ISSUE THREAD

**Issue by William-Purvis** [JuliaGraphics/Gtk.jl#381]
After finally getting Gtk.jl to work up to a point, I tried the 'get started' examples and 
managed to bring up a window, with a button. However, when I issued the 'signal_connect'
call, it doesn't seem to do anything. Clicking on the button fails to provoke the callback.

I'm running Linux Mint 18, Cinnamon if that helps.

**Comment by tknopp** [comment]
works for me. what do you mean by "Clicking on the button fails to provoke the callback." what is the error message?

**Comment by timholy** [comment]
It is a slightly misleading example in that calling `println` switches tasks, and really one shouldn't do that from within a callback. Better to make it `@async println("The button has been clicked")`.

**Comment by tknopp** [comment]
true. But it still works here, so I am wondering if there is another issue.

**Comment by William-Purvis** [comment]
No error message - the window responds by darkening while the button is dow, then reverts to
normal on release. Nothing appears in the command window. I add the @async but still nothing
printed.

**Comment by tknopp** [comment]
this is certainly related to #380.
